test/functional/rpc_getblockstats.py

```diff
@@ -55,13 +42,14 @@ def generate_test_data(self, filename):
         mocktime = time.time()
```

Comment:
```suggestion
        mocktime = 1525107225
```

nit: could set mocktime to the previous value, so that similar data is generated?